legal-api/src/legal_api/services/filings/validations/incorporation_application.py

```diff
@@ -94,17 +94,18 @@ def validate_offices(incorporation_json) -> Error:
     return None
 
 
+# pylint: disable=too-many-branches
 def validate_roles(incorporation_json) -> Error:
     """Validate the required completing party of the incorporation filing."""
     parties_array = incorporation_json['filing']['incorporationApplication']['parties']
+    legal_type = incorporation_json['filing']['incorporationApplication']['nameRequest']['legalType']
```

Comment:
If you are trying to get legalType of business get it from '/filing/business/legalType' and I would suggest to use get_str function

Reply:
updated to grab legal type from `/filing/business/legalType` and use `get_str` function